docker-compose.test.yml

```diff
@@ -17,7 +17,7 @@ services:
       - ${HOME}/.cache/go-build:/root/.cache/go-build
       - ${GOPATH}/pkg/mod:/go/pkg/mod
   db-unit-test:
-    image: postgres
+    image: postgres:13
```

Comment:
Just wanted to comment for anyone looking back at this PR.  The unit test DB relies on a `pgdump` from Postgres 13 DB that makes it very difficult to use that same dump in Postges 10.  Because it is simply the unit test DB, we determined it acceptable to keep this database at 13 while aligning the databases for the API with AWS.